src/client/cl_main.c

```diff
@@ -622,16 +618,19 @@ void Cl_Frame(const uint32_t msec) {
 
 	Cl_HandleEvents();
 
-	Cl_UpdateMovementCommand(frame_msec);
+	if (cls.state == CL_ACTIVE) {
 
-	Cl_SendCommands();
+		Cl_UpdateMovementCommand(frame_msec);
+
+		Cl_SendCommands();
 
-	if (cls.state == CL_ACTIVE) {
 		Cl_Interpolate();
 
 		Cl_PredictMovement();
 
 		Cl_UpdateView();
+	} else {
+		Cl_SendCommands();
 	}
```

Comment:
Simply updated `Cl_Frame` to not bother with `Cl_UpdateMovementCommand` until we're `CL_ACTIVE`.